server/channels/app/file.go

```diff
@@ -1390,7 +1407,12 @@ func (a *App) CreateZipFileAndAddFiles(fileBackend filestore.FileBackend, fileDa
 // This is a implementation of Go's example of writing files to zip (with slight modification)
 // https://golang.org/src/archive/zip/example_test.go
 func populateZipfile(w *zip.Writer, fileDatas []model.FileData) error {
-	defer w.Close()
+	defer func() {
+		if err := w.Close(); err != nil {
+			log.Println(err)
+		}
```

Comment:
Logging to stdout is not a great option. Let's add a `request.CTX` as the first argument to this function and it's caller and use the `rctx.Logger()` to log the error.

Reply:
Roger